File: Yusif/Day2-Challenges/Exercise5.js
// Write a program that adds the digits in a two-digit number.

// Example:

// 🔹 Input: 35

// 🔹 Output: 3 + 5 = 8

// 📝 Hint: Research how to convert data types in your programming language.
const readline =require("readline")

const rl=readline.createInterface({
    input:process.stdin,
    output:process.stdout
});
rl.question("Enter a two-didgit number: ",(num)=>{
if(num.length !==2 || isNaN(num)){
    console.log("please enter a valid two-digit number.");
}
else{
    const firstDigit=parseInt(num[0])
    const secondDigit=parseInt(num[1])
    const sum= firstDigit+secondDigit
    console.log(`${firstDigit} +${secondDigit}=${sum}`)
}
rl.close()
})

File: Pokua/BasicChallenges/Exercise5.java
package BasicChallenges;

import java.util.Scanner;

public class Exercise5 {
    public static void main (String[] args){
        Scanner input = new Scanner(System.in);

        System.out.println("Enter a two-digit number: ");
        String number = input.nextLine();
        String num1 = number.substring(0,1);
        String num2 = number.substring(1,2);
        int sumOfDigits =  Integer.parseInt(num1) + Integer.parseInt(num2);

        System.out.println("The sum of the two digits is: "+sumOfDigits);

        input.close();
    }
}
